### PR TITLE
Updated Docs link in badge

### LINKED ISSUE #53: Documentation badge doesn't point to helpful docs
The documentation badge currently points to the contents of `docs/`, which is just a page telling you how to compile the membrane curvature docs: https://www.mdanalysis.org/membrane-curvature/

Could you please change the link to ReadTheDocs, which is currently working documentation? https://membrane-curvature.readthedocs.io/en/latest/

### PR DESCRIPTION
**Pull request by ojeda-e**
Fixes #53 

Docs links updated to https://membrane-curvature.readthedocs.io/en/latest/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Membrane Curvature
...
 [![GitHub Actions Status](https://github.com/MDAnalysis/membrane-curvature/workflows/CI/badge.svg)](https://github.com/MDAnalysis/membrane-curvature/actions?query=workflow%3ACI)
 [![codecov](https://codecov.io/gh/MDAnalysis/membrane-curvature/branch/main/graph/badge.svg)](https://codecov.io/gh/MDAnalysis/membrane-curvature/branch/main)
-[![docs](https://readthedocs.org/projects/pip/badge/)](https://www.mdanalysis.org/membrane-curvature/)
+[![docs](https://readthedocs.org/projects/pip/badge/)](https://membrane-curvature.readthedocs.io/en/latest/)
```
Comment (lilyminium) [inline README.md:7]:
```suggestion
[![docs](https://readthedocs.org/projects/membrane-curvature/badge/?version=latest)](https://membrane-curvature.readthedocs.io/en/latest/)
```

My bad for not picking this up earlier, but I think it's actually picking up the build status of pip :p

**Review by lilyminium — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Membrane Curvature
...
 [![GitHub Actions Status](https://github.com/MDAnalysis/membrane-curvature/workflows/CI/badge.svg)](https://github.com/MDAnalysis/membrane-curvature/actions?query=workflow%3ACI)
 [![codecov](https://codecov.io/gh/MDAnalysis/membrane-curvature/branch/main/graph/badge.svg)](https://codecov.io/gh/MDAnalysis/membrane-curvature/branch/main)
-[![docs](https://readthedocs.org/projects/pip/badge/)](https://www.mdanalysis.org/membrane-curvature/)
+[![docs](https://img.shields.io/badge/docs-latest-brightgreen.svg)](https://membrane-curvature.readthedocs.io/en/latest/)
```
Comment (lilyminium) [inline README.md:7]: I'm not sure this is what you want, it looks like it's just always green. I think I pasted the right link earlier.

**Review by lilyminium — APPROVED** [review]
Thank you!

**Comment by ojeda-e** [comment]
> Thank you!

Thanks, @lilyminium, I am receiving delayed notifications from your comments and some of my replies weren't saved :s 
Nothing important, but just letting you know. I'll merge this. Thanks again :)

**Pull request merged**